geometric_kernels/basis.py

```diff
@@ -0,0 +1,219 @@
+"""
```

Comment:
What's the deal with this module? Doesn't seem to be referenced anywhere and `eigenfunctions.py` is still present